api/v2beta1/helmrelease_types.go

```diff
@@ -106,7 +105,7 @@ type HelmReleaseSpec struct {
 	// references to HelmRelease resources that must be ready before this HelmRelease
 	// can be reconciled.
 	// +optional
-	DependsOn []dependency.CrossNamespaceDependencyReference `json:"dependsOn,omitempty"`
+	DependsOn []meta.NamespacedObjectReference `json:"dependsOn,omitempty"`
```

Comment:
The comment above this still mentions `dependency.CrossNamespaceDependencyReference`.

Reply:
Fixed